core/src/main/resources/openapi-specs/eth2/signing/schemas.yaml

```diff
@@ -464,6 +465,19 @@ components:
       required:
         - version
         - block_header
+    BlockRequestCapella:
+      type: object
+      properties:
+        version:
+          type: string
+          enum:
+            - CAPELLA
```

Comment:
Annoying that we need to define this just for the enum value. But think we have to do this way with the current API.